test.py

```diff
@@ -21,11 +39,16 @@ def test_dim_order(order):
     compressed = compress(a, rate=8)
     recovered = decompress(compressed[0:16], (4, 4), np.dtype('float32'),
                            rate=8, order=order)
-    b = np.arange(16, dtype=np.float32).reshape((4, 4), order=order)
```

Comment:
Not sure what you mean? a is the full array, b is the truncated array. If you look at the fortran order array, it is not sequential because of the truncation so I can't just do C or F order on b.